openassessment/xblock/validation.py

```diff
@@ -373,6 +373,9 @@ def _inner(rubric_dict, assessments, leaderboard_show=0, submission_start=None,
         submission_dates = [(submission_start, submission_due)]
         assessment_dates = [(asmnt.get('start'), asmnt.get('due')) for asmnt in assessments]
         success, msg = validate_dates(oa_block.start, oa_block.due, submission_dates + assessment_dates, _)
+        if not success:
```

Comment:
Let's please add a wiki in ORA document all core changes in ORA, please follow the template which is mentioned in https://github.com/philanthropy-u/edx-platform/wiki/Core-Changes

## Title: This is the title of core change
This is a description of the core change
##### Why this core change is inevitable?
The reason why this core change is inevitable.
##### Why you can't move this change to `overrides`?
The reason why this core change can't be moved to overrides.
##### Link of the ticket?
[LP-XXXX](https://philanthropyu.atlassian.net/browse/LP-XXXX)
##### Link of the PR?
[PR](https://philanthropyu.atlassian.net/browse/LP-XXXX)
*****